io-engine/src/subsys/nvmf/subsystem.rs

```diff
@@ -75,6 +82,30 @@ impl Display for SubType {
 pub struct NvmfSubsystem(pub(crate) NonNull<spdk_nvmf_subsystem>);
 pub struct NvmfSubsystemIterator(*mut spdk_nvmf_subsystem);
 
+#[repr(C)]
+pub struct SpdkNvmfController(pub(crate) NonNull<spdk_nvmf_ctrlr>);
+
+impl SpdkNvmfController {
+    fn get_hostnqn(&self) -> String {
```

Comment:
In rust it's typical not to prefix getters with `get_`.

```suggestion
    fn hostnqn(&self) -> String {
```